src/ParallelAlgorithms/Events/ObserveErrorNorms.hpp

```diff
@@ -32,27 +33,24 @@
 namespace Frame {
 struct Inertial;
 }  // namespace Frame
-namespace Tags {
-struct Time;
-}  // namespace Tags
 /// \endcond
 
 namespace dg {
 namespace Events {
-template <size_t VolumeDim, typename Tensors, typename EventRegistrars>
+template <typename TemporalId, typename Tensors, typename EventRegistrars>
 class ObserveErrorNorms;
 
 namespace Registrars {
-template <size_t VolumeDim, typename Tensors>
+template <typename TemporalId, typename Tensors>
 struct ObserveErrorNorms {
```

Comment:
Since the non-type template parameters are gone, this can be just `using ObserveErrorNorms = Registration::Registrar<Events::ObserveErrorNorms, TemporalId, Tensors>;`